site/en/r2/guide/keras.ipynb

```diff
@@ -1203,18 +1219,16 @@
         "The following example distributes a `tf.keras.Model` across multiple GPUs on a\n",
```

Comment:
Could use a link to guide/distribute_strategy.ipynb right here "See the full guide"

Reply:
added